collect/dwr/cdec/queries.py

```diff
@@ -126,7 +126,7 @@ def get_station_data(station, start, end, sensors=[], duration=''):
     return get_raw_station_csv(station, start, end, sensors, duration)
 
 
-def get_raw_station_csv(station, start, end, sensors=[], duration='', filename=''):
+def get_raw_station_csv(station, start, end, sensors=[], duration='', filename=None):
```

Comment:
If adding filename here, should do the same for `get_station_data`, which is just an alias for `get_raw_station_csv`

Reply:
yup 6770e9d